x2gbfs/providers/cantamen.py

```diff
@@ -174,12 +174,16 @@ def _seats(self, attributes: List[str]) -> Optional[int]:
     def _filter_and_map_attributes(self, attributes: List[str], filter: List[str]) -> List[str]:
         filtered_attributes = []
         for attribute in attributes:
+            if self.attributes.get(attribute) is None:  # cantamen data error
+                continue
             if self.attributes[attribute] in filter:
                 filtered_attributes.append(self.attributes[attribute])
         return filtered_attributes
 
     def _is_stationwagon(self, attributes: List[str]) -> bool:
         for attribute in attributes:
+            if self.attributes.get(attribute) is None:  # cantamen data error
```

Comment:
I suggest to combine this with the following if, instead of accessing via `[]`,  `get()` will return None, the condition will not be true and the loop be continued also.

Reply:
This is what I did first, but lint-check did not accept my combined if statement:

            if self.attributes.get(attribute) in filter:
                filtered_attributes.append(self.attributes.get(attribute))

How would you formulate this if statement?

Reply:
In the if condition, you can retrieve via get, when appending, the attribute exists and you may access via [].

This linted successfully:
```python
if self.attributes.get(attribute) in filter:
      filtered_attributes.append(self.attributes[attribute])
```